vc17/otclient.vcxproj

```
@@ -690,7 +690,7 @@
   </Target>
   <PropertyGroup Condition="'$(RunProtoCompile)'=='true'">
     <ProtocPath>$(ProjectDir)vcpkg_installed\$(VcpkgTriplet)\tools\protobuf\protoc</ProtocPath>
-    <ProtoPath>..\$(SourcePath)\protobuf</ProtoPath>
+    <ProtoPath>$(SourcePath)protobuf</ProtoPath>
```

It didn't compile in vs solution, I hope this doesn't break github actions or cmake
```suggestion
    <ProtoPath>$(SourcePath)protobuf</ProtoPath>
```